### PR TITLE
BXMSDOC-8490-main: Add step about ImageRegistry and image usage for KieApp CR

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

**Review by amadhusu — APPROVED** [review]
It looks perfect @csherrar. Approved.

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+|Description
+|*Image context*
+|The context of the image in the registry.
+|*Image*
+|The name of the image.
+|*Image tag*
+|The tag of the image. If you do not set this field, the installation uses the latest tag.
+|===
+
+* You want to specify the image by using a KieApp CR YAML file, add the the image registry and image details to the file, for example:
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:28]: The "If" at the start might have been lost.

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+       ...
+       image: YOUR_IMAGE_NAME
+       imageContext: YOUR_IMAGE_CONTEXT
+       imageTag: YOUR_IMAGE_TAG
+       ...
+----
++
+[NOTE]
+====
+The operator does not create an Imagestream if there is an existing imagestream in the namespace.
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:54]: I would suggest making this phrase more positive. Something like:

If an ImageStream with the configured name exists in the namespace that you are using or in the OpenShift namespace, the operator uses this ImageStream and does not create a new ImageStream.

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+The operator also does not create an Imagestream if the samples operator created the imagestreams in the OpenShift namespace.
+
+To configure an Imagestream to update automatically, the `scheduledImportPolicy` property must be set to `true`.
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:58]: Is this a property of the ImageStream?

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+       imageContext: YOUR_IMAGE_CONTEXT
+       imageTag: YOUR_IMAGE_TAG
+       ...
+----
++
+[NOTE]
+====
+The operator does not create an Imagestream if there is an existing imagestream in the namespace.
+
+The operator also does not create an Imagestream if the samples operator created the imagestreams in the OpenShift namespace.
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:56]: I would reformulate this to kinda refer to the previous paragraph?

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+During the basic configuration of the environment you can configure the custom registry to use specific images. For more information about configuring the basic environment, see <<operator-deploy-basic-proc_{context}>>.
+
+.Prerequisites
+* You started to configure a {PRODUCT} environment using {operator_name} and accessed the installer wizard according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+* You have set the URL of the image registry in the *Image registry* field under the *Custom registry* of the {PRODUCT} environment configuration according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+
+.Procedure
+
+. During the {PRODUCT} environment configuration, to use a specific image from the custom registry, perform one of the following tasks:
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:12]: Done

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:28]: Done

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc]: It is a propoerty of the ImageStream. I updated the content to reflect this. "To configure an ImageStream to update automatically, the ImageStream property `scheduledImportPolicy` must be set to `true` in the KieApp CR YAML file."

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc]: Updated to reflect your feedback.
"If an ImageStream with the configured name exists in the namespace that you are using or in the {OPENSHIFT} namespace, the operator uses this ImageStream and does not create a new ImageStream."

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+During the basic configuration of the environment you can configure the custom registry to use specific images. For more information about configuring the basic environment, see <<operator-deploy-basic-proc_{context}>>.
+
+.Prerequisites
+* You started to configure a {PRODUCT} environment using {operator_name} and accessed the installer wizard according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+* You have set the URL of the image registry in the *Image registry* field under the *Custom registry* of the {PRODUCT} environment configuration according to the instructions in <<operator-deploy-basic-proc_{context}>>.
+
+.Procedure
+
+* During the {PRODUCT} environment configuration, to use a specific image from the custom registry, perform one of the following tasks:
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:12]: I'd use "complete one of the following steps:"

File: doc-content/enterprise-only/openshift/operator-environment-deploy-assy.adoc [hunk 7]
```diff
@@ -11,6 +11,7 @@ In {PRODUCT} {PRODUCT_VERSION}, the operator installer wizard is for Technology
 
 include::operator-deploy-start-proc.adoc[leveloffset=+1]
 include::operator-deploy-basic-proc.adoc[leveloffset=+1]
+include::configuring-image-registry-images-proc.adoc[leveloffset=+1]
```
Comment (mramendi) [inline doc-content/enterprise-only/openshift/operator-environment-deploy-assy.adoc:14]: I suggest using leveloffset=+2

**Review by mramendi — APPROVED** [review]

File: doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/configuring-image-registry-images-proc.adoc:12]: Done

File: doc-content/enterprise-only/openshift/operator-environment-deploy-assy.adoc (hunk 7 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-environment-deploy-assy.adoc]: Done

**Pull request merged**